com/abweichung.go

```diff
@@ -10,6 +10,8 @@ type Abweichung struct {
 	AbweichungsGrundBemerkung *string                            `json:"abweichungsgrundBemerkung,omitempty"`
 	AbweichungsgrundCode      *string                            `json:"abweichungsgrundCode,omitempty" validate:"omitempty"`
 	AbweichungsgrundCodeliste *string                            `json:"abweichungsgrundCodeliste,omitempty" validate:"omitempty"`
+	ZugehoerigeRechnung       *string                            `json:"zugehoerigeRechnung,omitempty" validate:"omitempty"`
+	Abschlagsrechnungen       []string                           `json:"abschlagsrechnungen,omitempty" validate:"omitempty"`
```

Comment:
https://github.com/Hochfrequenz/BO4E-dotnet/blob/44559f157bd98232bac8ae75583def0c6e057683/BO4E/COM/Abweichung.cs#L54

diese liste ist in c# nullable. kommt go damit klar?

Reply:
arrays sind per se nullable, weil sie nur ansammlungen an pointern sind